src/utils/Processing.ts

```diff
@@ -112,7 +113,26 @@ export function writeFSH(resources: Package, outDir: string, style: string): voi
   }
 }
 
-export function loadExternalDependencies(
+export async function loadExternalDependencies(
+  defs: FHIRDefinitions,
+  config: ExportableConfiguration
+) {
+  const allDependencies =
+    config.config.dependencies?.map(
+      (dep: fhirtypes.ImplementationGuideDependsOn) => `${dep.packageId}@${dep.version}`
+    ) ?? [];
+  const fhirPackageId = determineCorePackageId(config.config.fhirVersion[0]);
+  allDependencies.push(`${fhirPackageId}@${config.config.fhirVersion[0]}`);
```

Comment:
I'm actually not sure which way is what we want, but I noticed that in the old `fhirToFsh` function, we only pushed on the FHIR version if it wasn't already included in the dependency list. I think it makes sense to have that logic here, but I also noticed we did _not_ have that logic in `app.ts`. It might be good to avoid trying to load that package twice, but I'm curious what you think.

Reply:
I think it makes sense to not try to load the package twice, so I'll add in that check.